polkadot-parachains/integritee-runtime/Cargo.toml

```diff
@@ -11,95 +11,94 @@ edition = "2021"
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
 log = { version = "0.4.17", default-features = false }
-parachain-info = { default-features = false, git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.29" }
-scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
-serde = { version = "1.0.140", default-features = false, optional = true, features = ["derive"] }
+scale-info = { version = "2.3.1", default-features = false, features = ["derive"] }
```

Comment:
Serde was an unused dependency. Parachain Info has been put into the correct group of dependencies below.